### PR TITLE
Feature/sponsored events

### PR DESCRIPTION
**Pull request by markkohdev**
Handle sponsored events so people don't input dumb stuff during spring jamz.

I also fixed a buncha stuff and cleaned up the gross code that was the EnterConcertInformationController.

Sweet dolla tea from McDonalds

### CODE REVIEW

File: JamJar/AppDelegate.swift [hunk 1]
```diff
@@ -28,6 +28,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+        // Uncomment to print all fonts
+        //        let fontFamilyNames = UIFont.familyNames()
+        //        for familyName in fontFamilyNames {
+        //            print("------------------------------")
+        //            print("Font Family Name = [\(familyName)]")
+        //            let names = UIFont.fontNamesForFamilyName(familyName as! String)
+        //            print("Font Names = [\(names)]")
+        //        }
+        
         return true
```
Comment (ethan-riback) [inline JamJar/AppDelegate.swift:41]: What is this for?

File: JamJar/EnterConcertInformationController.swift [hunk 2]
```diff
@@ -32,8 +32,16 @@ class EnterConcertInformationViewController: BaseViewController, UITextFieldDele
...
+    @IBOutlet weak var sponsoredStackView: UIStackView!
     @IBOutlet weak var artistsStackView: UIStackView!
     
+    
+    /***************************************************************************
+     Loading up a buncha stuff
+        Primarily the callbacks and stuff for the AutoComplete inputs
+        (should definitely be cleaned up eventually at some point maybe perhaps)
+     ***************************************************************************/
+
```
Comment (ethan-riback) [inline JamJar/EnterConcertInformationController.swift:45]: Perhaps....

File: JamJar/EnterConcertInformationController.swift [hunk 3]
```diff
@@ -165,14 +286,24 @@ class EnterConcertInformationViewController: BaseViewController, UITextFieldDele
...
     }
+
+
+    /***************************************************************************
+     Concert Date Stuff
+     ***************************************************************************/
     
-    func dataPickerChanged(sender:UIDatePicker) {
+    func datePickerChanged(sender:UIDatePicker) {
         // Make the date text field human-readable
```
Comment (ethan-riback) [inline JamJar/EnterConcertInformationController.swift:299]: Did something change here? I am a bit confused....

**Comment by ethan-riback** [comment]
If a current event is selected that has no artists, any previously selected artists will remain. I do not know if this was an intentional design choice or not.

**Comment by markkohdev** [comment]
MERGEEEEEEEIN 🍠 🍯 🚰

**Pull request merged**